app/domain/authentication/authn_jwt/signing_key/signing_key_settings_builder.rb

```diff
@@ -0,0 +1,133 @@
+module Authentication
+  module AuthnJwt
+    module SigningKey
+
+      NO_SIGNING_KEYS_SOURCE = "One of #{JWKS_URI_RESOURCE_NAME}, #{PUBLIC_KEYS_RESOURCE_NAME}, and #{PROVIDER_URI_RESOURCE_NAME} have to be defined".freeze
```

Comment:
Can it sit in consts or errors.rb?

Reply:
It's too specific from my point of view.
Nobody else is using the same coasts so there's no reason to split it to other files.
As I mentioned before errors.rb is a container for exceptions, not for error messages.